tests/user.test.js

```diff
@@ -2,6 +2,7 @@
 
 const supertest = require('supertest');
 const { createServer } = require('../src/createServer');
+const { clearUsers } = require('../src/services/user.service');
 
 describe('User', () => {
```

Comment:
It seems you are missing a cleanup function after each test. This is important to ensure that the tests do not affect each other and that the database is in a known state before each test. In Jest, you can use the afterEach function for this. For example: `afterEach(async () => { await clearUsers(); });`